pkg/directorysync/client.go

```diff
@@ -137,22 +137,22 @@ type User struct {
 // ListUsersOpts contains the options to request provisioned Directory Users.
 type ListUsersOpts struct {
 	// Directory unique identifier.
-	Directory string
+	Directory string `url:"directory", omitempty`
 
 	// Directory Group unique identifier.
-	Group string
+	Group string `url:"group", omitempty`
```

Comment:
struct tags are weird magic strings but they are pretty picky about the format.

```suggestion
	Group string `url:"group,omitempty"`
```
(this is the case with a decent number of the tags, just commenting on this one though!)